packages/experiments/src/components/Tile/Tile.tsx

```diff
@@ -86,6 +89,8 @@ export class Tile extends BaseComponent<ITileProps, ITileState> {
       isSelected: isSelected,
       isModal: isModal
     };
+
+    this._events = new EventGroup(this);
```

Comment:
Do we really need EventGroup? Could you use `on.ts` from utilities instead? See usage.

Reply:
Not sure if it applies in this case but as FYI, anything using DragDropHelper must keep using EventGroup for now. 

Also, it would be nice to have a better pattern or utility for tracking and disposing `on` handlers. Currently I find the pattern cumbersome.

Reply:
> Do we really need EventGroup? Could you use `on.ts` from utilities instead? See usage.

Currently, selection events are bound to an object rather than `Element | Window` and the `on` helper assumes a DOM node is being un/subscribed to. Its a worthwhile consideration as part of these refactors, but I think its reasonable to continue to use `EventGroup` in this PR.

> Also, it would be nice to have a better pattern or utility for tracking and disposing `on` handlers. Currently I find the pattern cumbersome.

Would the hooks effect pattern be applied to that helper in terms of disposal? If so that would be rad.

Reply:
FYI `List` requires `EventGroup` for the time being. It handles a lifecycle issue with the scrollable surface that `on` does not which was caught by unit tests.